src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -97,7 +99,10 @@ private Integer getNumPartitions(String topic) {
 	}
 
 	private String generateTopic(String topic, RowMap r){
```

Comment:
yeah.   but it doesn't matter.  And in many cases anything you do to make something more of a "pure" function is better.